src/com/geekhub/hw3/comparable/ArraySorter.java

```diff
@@ -18,7 +18,19 @@ public static void main(String[] args) {
      * @return sorted array
      */
     public static Comparable[] sort(Comparable[] elements) {
```

Comment:
It will be good to create some Class (e.g Person) which implements Comparable interface. And try to sort array of these objects.